rust/agama-server/src/agama-web-server.rs

```diff
@@ -239,6 +239,8 @@ async fn start_server(address: String, service: Router, ssl_acceptor: SslAccepto
 
     pin_mut!(listener);
 
+    println!("Agama web server listening at: {}", address);
```

Comment:
I'd remove this. It might mess the console when starting automatically via systemd. The address is printed by the tracing call above.